routes/index.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  res.render('index', { title: 'Express' });
```

Comment:
When you remove `pug`, this will cause an error, so either delete this file or remove the `res.render()` bit, and replace it with `res.redirect('/')` or something similar.